packages/wonder-blocks-popover/package.json

```diff
@@ -18,6 +18,8 @@
   "dependencies": {
     "@khanacademy/wonder-blocks-color": "^1.1.4",
     "@khanacademy/wonder-blocks-core": "^2.3.0",
+    "@khanacademy/wonder-blocks-icon": "^1.2.2",
```

Comment:
All these dependencies were missing - or were they only used in the docs? If they're only in the docs then adding them to `devDependencies` would probably be better.

Reply:
They were missing dependencies. I'm using them to create the CloseButton component.